configs/setups/awicm3/awicm3.yaml

```diff
@@ -30,7 +34,7 @@ general:
             add_include_models:
             - xios
 
-        version: "1.0-deck"
+        versions: "1.0-deck"
```

Comment:
We need to define a default `version`. Which version is entirely up to you, but a default needs to be given, otherwise this can result into problems for ESM-Tools I think

Reply:
👍